docs/topics/native/native-cocoapods-libraries.md

```diff
@@ -257,6 +262,7 @@ import cocoapods.example.*
             ios.deploymentTarget = "13.5"
 
             pod("YandexMapKit") {
+                extraOpts = listOf("-compiler-option", "-fmodules")
```

Comment:
It is not required for this library, so I'm not sure if it is a good idea add it here.